<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-HO0UVJN8" failures="1" tests="10" name="Test" time="14.478" errors="0" timestamp="2025-03-11T00:02:32 CET">
  <testcase classname="testCases.TC001_VerifyLoginFunctionality" name="verify_LoginCredentials" time="0.655"/>
  <testcase classname="testCases.TC001_VerifyLoginFunctionality" name="verify_LoginCredentials" time="0.406"/>
  <testcase classname="testCases.TC001_VerifyLoginFunctionality" name="verify_LoginCredentials" time="0.36"/>
  <testcase classname="testCases.TC001_VerifyLoginFunctionality" name="verify_LoginCredentials" time="0.587"/>
  <testcase classname="testCases.TC001_VerifyLoginFunctionality" name="verify_LoginCredentials" time="0.551"/>
  <testcase classname="testCases.TC001_VerifyLoginFunctionality" name="verify_LoginCredentials" time="0.329"/>
  <testcase classname="testCases.TC001_VerifyLoginFunctionality" name="verify_LoginCredentials" time="5.746"/>
  <testcase classname="testCases.TC001_VerifyLoginFunctionality" name="verify_LoginCredentials" time="0.687"/>
  <testcase classname="testCases.TC001_VerifyLoginFunctionality" name="verify_LoginCredentials" time="0.878"/>
  <testcase classname="testCases.TC001_VerifyLoginFunctionality" name="verify_LoginCredentials" time="0.107">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC001_VerifyLoginFunctionality.verify_LoginCredentials(TC001_VerifyLoginFunctionality.java:82)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_LoginCredentials -->
</testsuite> <!-- Test -->
